### DISCUSSION TITLE
Why is type of page break set to oddPage in landscape tags?

### DISCUSSION THREAD

**Discussion by PatrickRWright** [davidgohel/officedown#138] [Q&A] [answered]
The documentation (https://ardata-fr.github.io/officeverse/officedown-for-word.html#tags-for-sections) states:  
"The type of page break is set to `oddPage`."

To me it seems like this default setting is creating unexpected output. I would expect it to be set to `"continuous"`. 

Presumably I can change it like so when stopping a landscape setup:  
`<!---BLOCK_LANDSCAPE_STOP{break_page: "continuous"}--->`

However, I'm not sure if this has unexpected consequences since the default may have been chosen based on some rationale. What's the exact reasoning for the choice of the default?

Part of this is covered in this StackOverflow question that I have asked (just as an FYI for a reproducible example):  
https://stackoverflow.com/questions/78237542/why-are-landscape-pages-causing-issues-with-page-numbers-section-headers-footers

**Comment by davidgohel** [comment, marked as answer]
Because there must be a value. When using "nextPage", some users will complain and argue "it should be OddPage", when using "OddPage", some users will complain and argue "it should be "nextPage". I think if we use landscape orientation, it involves the odd and the even page (think about a printed version), but it could be argued that landscape orientation could start on page 3 and end on page 4. 

From https://ardata-fr.github.io/officeverse/officedown-for-word.html#function-block_section

> If the default settings associated with the tags don’t satisfy your needs, you can define a section ending with a call to function block_section within a knitr R chunk.

So you can change the default value with a call to `block_section()`


> I would expect it to be set to "continuous".

A page can not be landscape oriented starting in its middle position

**Reply by PatrickRWright** [reply]
Ok I guess I'm one of the `"nextPage"` users. `"continuous"` seemed valid because - at least at first glance in my example - it seems to be achieving the same result as setting `"nextPage"`.

The "printed version" argument never even occurred to me but I get the point and see its validity depending on whom you may ask.

Using `<!---BLOCK_LANDSCAPE_STOP{break_page: "nextPage"}--->` seems more concise to me as oppsed to adding further chunks. I assume its functionally equivalent to:
````
```{r}
block_section(prop_section(type = "nextPage"))
```
````

Thanks for your time!
